Source/CMake/Plugins/BsImgui.cmake

```diff
@@ -0,0 +1,45 @@
+include(FetchContent)
+
+
+# if INCLUDE_IMGUI_PLUGIN is true, then git clone the imgui plugin (with
+# recursive submodules). it will be cloned into the build directory where it
+# will be configured as bsfImgui target. other targets can then set bsfImgui
+# as their dependency in their respective cmake files.
+
+set(FETCHCONTENT_QUIET CACHE INTERNAL 0)
+
+FetchContent_Declare(
+  imgui
```

Comment:
Where does this fetch to (i.e. what folder)?

Reply:
it will fetch to the Build/_deps/imgui-src folder and is built in the Build/_deps/imgui-build folder.